### ISSUE TITLE
Add url / links

### ISSUE THREAD

**Issue by smellymoo** [ArsMasiuk/qvge#174]
It would be super useful if you could link to files and websites using URLs, specially linking to other QVGE files.

**Comment by ArsMasiuk** [comment]
What do you mean? Open files by drag&drop?

сб, 21 окт. 2023 г. в 11:08, smellymoo ***@***.***>:

**Comment by smellymoo** [comment]
no, I mean having URLs supported, so that you can link to other files / graphs / websites.

So the text at the center of a bubble could take you to the browser for websites, and for graph files it could open them directly.
